src/libexpr/primops.cc

```diff
@@ -1691,13 +1691,14 @@ static void prim_findFile(EvalState & state, const PosIdx pos, Value * * args, V
 
 static RegisterPrimOp primop_findFile(PrimOp {
     .name = "__findFile",
-    .args = {"search path", "lookup path"},
+    .args = {"search-path", "lookup-path"},
     .doc = R"(
-      Look up the given path with the given search path.
+      Find a *lookup-path* in the *search-path*.
```

Comment:
```suggestion
      Find *lookup-path* in *search-path*.
```

This makes *lookup-path* look to me like some sort special term needing a glossary entry, but I don't think that is our intention. I think it is just the name you are using for the algorithm description below.

Reply:
I had this originally, and while technically correct it feels like really distracting to read. Yes, "the" search path may be a bit off, but really within an evaluation run there is only one, while there may be multiple lookup paths.

Reply:
You can use a custom search path:

```bash
$ nix-instantiate --expr 'builtins.findFile [{ prefix = "foo"; path = "/home"; }] "foo"' --eval
/home
```